app/views/registrations/_userAgreement.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="row">
```

Comment:
Die Namen der Dateien werden in Rails nicht `_userAgreement.html.erb` sondern wie folgt geschrieben: `_user_agreement.html.erb`

Reply:
wurde geändert